pages/entropy/debug-callback-failures.mdx

```diff
@@ -0,0 +1,91 @@
+# Debug Callback Failures
+
+This guide explains how to identify and resolve issues with the Entropy callback.
+The intended audience for this guide is developers who have made an Entropy random number request, but their application hasn't received a callback.
+
+## Dependencies
+
+This guide uses [Foundry](https://book.getfoundry.sh/getting-started/installation) to submit transactions to the blockchain.
+Please install Foundry before continuing.
+
+## Run the Callback
+
+Developers can run the Entropy callback themselves to see the reason for the failure.
+To run the callback, invoke the `revealWithCallback` function on the Entropy contract on your blockchain.
+The function has the following signature:
+
+```solidity
+function revealWithCallback(address provider, uint64 sequenceNumber, bytes32 userRandomNumber, bytes32 providerRevelation)
+```
+
+This call requires the chain id, contract address and four arguments.
+The chain id and contract address can be retrieved from [Contract Addresses](/entropy/contract-addresses).
+Export these values as environment variables for later use:
+
+```bash
+export CHAIN_ID=blast
+export ENTROPY_ADDRESS=0x5744Cbf430D99456a0A8771208b674F27f8EF0Fb
+```
+
+Three of the arguments can be retrieved from the request transaction's event logs.
```

Comment:
Shouldn't we give a small explanation here about these arguments?

Reply:
I'm trying to stick to the diataxis guideline to avoid explanation in how-to guides. I also think anyone reading this guide will know what these are (because they wrote the code to integrate the software in the first place)